operator/v1/types_console.go

```diff
@@ -143,8 +143,42 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomFiles defines a configuration based on theme types for the console UI logo.
+type CustomFiles struct {
```

Comment:
A couple things here:
- the name `CustomFiles` is not something that I would generally associate with a configuration for a logo based on themes. Is there another name we could use for the struct that is more representative of this being a theme-based logo substitution (or at the least a logo substitution)?
- This is pluralized but represents a single object. I would avoid pluralization of a type unless it is aliased to something like a slice or map.